src/components/add-tag-modal/form/index.js

```diff
@@ -17,6 +17,9 @@
 */
 import React, { PropTypes } from 'react';
 
+import { Geotags } from '../deps';
+import { Hashtags } from '../deps';
```

Comment:
it is a very clear abstraction leak. A change upstream will lead to immediate break here. Which means that this component is not really a component. Which means that the whole point of `deps` just complicates matters here. Either it should aim to be independent and we're planning to extract it into separate package or let's forget about `deps`.